_drafts/partially-ordering-time.md

```diff
@@ -0,0 +1,253 @@
+# Time Is Weird
+## or: why do distributed systems and weak memory models look so similar, anyway
+
+(_Epistemic status: I'm just learning about both of these fields, and these are simply my
+observations from studying them and comparing them. If you work in either field and
+something looks wrong, it probably is; let me know!_)
+
+    There's only one hard problem in computer science: recognising that cache invalidation errors
+    are misnamed. They're just off-by-one errors in the time domain.
+        —Unknown
+
+Time is weird.
+
+Time is weird, not just because we insist on trying to [measure it](#url) and [standardise
+it](#url), though that field is of course amazingly horrifying. Time is weird, because
+when we try to think about what it actually _implies_, that a second has passed since one
+second ago.
```

Comment:
> when we try to think about what it actually _implies_, that a second has passed since one second ago.

I would use a colon here, and I feel like it'd be better to add just a bit more explanation to develop the weirdness of "a second ago":

> when we try to think about what it actually _implies_: that 1 second has passed from a "previous now" to a "current now". Of course, in the time it took to think that, the "previous now" was likely _many seconds ago_.

Something to that effect. I tried to provide an example in a similar tone. If you specifically are trying to get the reader to really simmer on the idea of "a second has passed since one second ago," I think it is worth while to somehow develop/define the concept of time passing with respect to some notion of "now," which I assume is what one of the things you're trying to refer to when you say "time is weird" in this context.